src/generate.js

```diff
@@ -129,6 +143,11 @@ function generate() {
     mkdirp('public');
     fs.writeFileSync('public/index.html', indexHtml);
 
+    copyFiles([
```

Comment:
Probably a bit too complicated: check the use of `copyFileSync` elsewhere in this file.